site/docs/master/restic.md

```diff
@@ -259,21 +218,15 @@ and `restic prune`.
 
     You can see information about your Velero restic repositories by running `velero restic repo get`.
 
-- `PodVolumeBackup` - represents a restic backup of a volume in a pod. The main Velero backup process creates
-one or more of these when it finds an annotated pod. Each node in the cluster runs a controller for this
-resource (in a daemonset) that handles the `PodVolumeBackups` for pods on that node. The controller executes
-`restic backup` commands to backup pod volume data.
+- `PodVolumeBackup` - represents a restic backup of any volume in a pod. The main Velero backup process creates
```

Comment:
I don't think this needs to be edited, given that we're still using pod annotations to indicate the desire to back up a specific pod volume.